atomic_scanners/scanner-rpm-verify/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM docker.io/centos:latest
```

Comment:
Use the image `registry.centos.org/centos/centos` instead. It would speed image creation as well.